docs/admin/installation/hardware.rst

```diff
@@ -75,7 +75,12 @@ and recycled ThinkPad laptops work well for the
 *Admin Workstation*/*Journalist Workstation*.
 
 Please note that very old laptops or desktops may not work for the workstations.
-Since the release of Tails 3.0, 32-bit computers are no longer supported.
+Since the release of Tails 3.0, 32-bit computers are no longer supported.asdfasdf
```

Comment:
> asdfasdf

I've been there. ;-)

Reply:
Not sure whether to laugh or cry :upside_down_face:

Starting to think this PR is cursed!